MANIFEST.in

```diff
@@ -9,7 +9,9 @@ include demo/*.py
 include demo/*.sh
 
 recursive-include synapse/storage/schema *.sql
+recursive-include synapse/storage/schema *.sql.*
 recursive-include synapse/storage/schema *.py
+recursive-include synapse/storage/schema *.md
```

Comment:
We don't seem to have added any of these sorts of files?

Reply:
ah, that was from when I had the full schema (which I pulled out of this PR). Will remove.